packages/xo-web/src/common/intl/messages.js

```diff
@@ -1662,6 +1662,11 @@ const messages = {
     'Are you sure you want to delete all the backups from {nMetadataBackups, number} metadata backup{nMetadataBackups, plural, one {} other {s}}?',
   bulkDeleteMetadataBackupsConfirmText:
     'delete {nMetadataBackups} metadata backup{nMetadataBackups, plural, one {} other {s}}',
+  healthCheck: 'Health check',
+  healthCheckSr: 'Choose SR used for VMs restoration',
+  healthCheckTags: 'VM tags',
```

Comment:
It may be used more than once.
```suggestion
  vmsTags: 'VMs tags',
```